TeslaSUI/TeslaSUI/Views/ClimateView.swift

```diff
@@ -0,0 +1,393 @@
+                fanClimateSettingView
+                heatClimateSettingView
+                autoClimateSettingView
+            }
+        } label: {
+            HStack {
+                Text(Constants.Text.climateSettings)
+                    .font(.system(size: 17, weight: .semibold))
+                    .foregroundColor(.white)
+            }
+        }
+        .accentColor(.white)
+        .padding()
+    }
+    
+    private var acClimateSettingView: some View {
+        HStack {
+            Text(Constants.Text.ac)
+                .font(.system(size: 17, weight: .semibold))
+                .frame(width: 50)
+            Button {
+                climateViewModel.isAcButtonTapped.toggle()
+            } label: {
+                Image(Constants.Images.snowIconImageName)
+                    .resizable()
+                    .modifier(ButtonImageModifire())
+            }
+            .neumorphismUnSelectedCircleStyle()
+            .overlay(
+            Circle()
+                .stroke(gradient, lineWidth: 2)
+                .opacity(climateViewModel.isAcButtonTapped ? 1 : 0)
+            )
+            
+            Slider(value: $climateViewModel.circleProgress,
+                   in: Constants.Integer.sliderRange,
+                   step: 1)
+            .frame(width: 220)
+        }
+    }
+    
+    private var fanClimateSettingView: some View {
+        HStack {
+            Text(Constants.Text.fan)
+                .font(.system(size: 17, weight: .semibold))
+                .frame(width: 50)
+            Button {
+                
+            } label: {
+                Image(Constants.Images.fanIconImageName)
+                    .resizable()
+                    .modifier(ButtonImageModifire())
+            }
+            .neumorphismUnSelectedCircleStyle()
+            
+            Slider(value: $climateViewModel.fanParameters,
+                   in: Constants.Integer.sliderRange,
+                   step: 1)
+            .frame(width: 220)
+        }
+    }
+    
+    private var autoClimateSettingView: some View {
+        HStack {
+            Text(Constants.Text.auto)
+                .font(.system(size: 17, weight: .semibold))
+                .frame(width: 50)
+            Button {
+                
+            } label: {
+                Image(Constants.Images.autoIconImageName)
+                    .resizable()
+                    .modifier(ButtonImageModifire())
+            }
+            .neumorphismUnSelectedCircleStyle()
+            Slider(value: $climateViewModel.autoParameters,
+                   in: Constants.Integer.sliderRange,
+                   step: 1)
+            .frame(width: 220)
+        }
+    }
+    
+    private var heatClimateSettingView: some View {
+        HStack {
+            Text(Constants.Text.heat)
+                .font(.system(size: 17, weight: .semibold))
+                .frame(width: 50)
+            Button {
+                
+            } label: {
```

Comment:
```suggestion
            Button {} label: {
```
почему то не везде отформатировалось, проверь везде где нет экшена в баттоне